src/tools/rbd/Utils.cc

```diff
@@ -832,6 +832,10 @@ int get_formatter(const po::variables_map &vm,
                 << "is json or xml" << std::endl;
       return -EINVAL;
     }
+  } else if (vm[at::PRETTY_FORMAT].as<bool>()) {
```

Comment:
Seems `vm[at::PRETTY_FORMAT].count()` (or maybe I missed something in it) is wrong when compiling and `vm.count(at::PRETTY_FORMAT)` always equal to 1, I used `vm[at::PRETTY_FORMAT].as<bool>() > 0`.  
Others have been updated, thanks for your review.